src/components/Home.js

```diff
@@ -1,10 +1,26 @@
-import React from 'react';
 
-const Home = () => {
+import React from "react";
```

Comment:
Prettier should auto-format quotes to be single, not double. Do you have the VS Code Prettier extension installed?

Reply:
Yes, it is installed, however I am not sure if it automatically format the quotes.